examples/federated_statistics/README.md

```diff
@@ -0,0 +1,47 @@
+# Objective
+Federated Statistics will provide pre-built-in NVFLARE federated statistics operators ( controller and executors) that 
+generate global statistics based on local client side statistics.
+
+At each clide site, we could have one more datasets (such as "train" and "test" datasets); each datasets may have many 
+features. For each named dataset, for each feature in the dataset, we will calculate the statistics and combined to produce 
+global statistics for the combined features. 
+
+These common used statistics are:  count, sum, mean, std_dev and histogram for numerical data.
+
+If the metrics sum and counts are selected, the mean will be calculate to use count and sum. 
```

Comment:
If the metric sum and count are selected, the mean will be calculated to use the count and sum.